tests/v2/validation/publicapi/clusterroletemplatebindings/crtb_create_steve_test.go

```diff
@@ -0,0 +1,307 @@
+	clusterList, err := client.Steve.SteveType(clusters.ProvisioningSteveResourceType).ListAll(nil)
+	require.NoError(t, err)
+	var clusterName string
+	for _, clusterData := range clusterList.Data {
+		if clusterData.JSONResp["metadata"] != nil {
+			metadata, ok := clusterData.JSONResp["metadata"].(map[string]interface{})
+			if ok {
+				if nameValue, exists := metadata["name"]; exists {
+					if nameStr, isString := nameValue.(string); isString && nameStr != "local" {
+						clusterName = nameStr
+						break
+					}
+				}
+			}
+		}
+	}
+	return clusterName
+}
+
+func createJsonTemplateForClusterRoleTemplateBinding() CRTBGenTestSuite {
+	// Create a JSON object for ClusterRoleTemplateBinding template
+	crtbTemplate := map[string]interface{}{
+		"type":       "clusterRoleTemplateBinding",
+		"apiVersion": "management.cattle.io/v3",
+		"kind":       "ClusterRoleTemplateBinding",
+		"metadata": map[string]interface{}{
+			"namespace": "",
+			"name":      "",
+			"labels": map[string]interface{}{
+				"hello": "World",
+			},
+		},
+		"clusterName":       "",
+		"roleTemplateName":  "",
+		"userName":          "",
+		"userPrincipalName": "",
+	}
+
+	jsonTemplateData, _ := json.Marshal(crtbTemplate)
+
+	return CRTBGenTestSuite{jsonData: jsonTemplateData}
+}
+
+func unmarshalReadJson(t *testing.T, readJson []byte, remoteClusterID string, crtbName string, newUser string, roleTemplateName string) map[string]interface{} {
+	// Unmarshal JSON object for ClusterRoleTemplateBinding template
+	var inputData map[string]interface{}
+	err := json.Unmarshal(readJson, &inputData)
+	require.NoError(t, err)
+	inputData["metadata"].(map[string]interface{})["namespace"] = remoteClusterID
+	inputData["clusterName"] = remoteClusterID
+	inputData["metadata"].(map[string]interface{})["name"] = crtbName
+	inputData["userName"] = newUser
+	inputData["roleTemplateName"] = roleTemplateName
+	return inputData
+}
+
+func validateCrtbCreationBySteveWithAllRequiredInputValues(t *testing.T, client *rancher.Client, remoteClusterID string, roleTemplateName string, readJson []byte) {
+	newUser, err := users.CreateUserWithRole(client, users.UserConfig())
+	require.NoError(t, err)
+	crtbName := namegen.RandStringLower(8)
+
+	inputData := unmarshalReadJson(t, readJson, remoteClusterID, crtbName, newUser.ID, roleTemplateName)
+	require.NoError(t, err)
+	resp, err := client.Steve.SteveType(crtbApiEndPoint).Create(inputData)
+	require.NoError(t, err)
+	assert.Contains(t, (*resp).JSONResp["id"], crtbName)
+}
+
+func validateCrtbCreationBySteveWithServiceAccountAsInputAndItIsIgnored(t *testing.T, client *rancher.Client, remoteClusterID string, readJson []byte) {
+	newUser, err := users.CreateUserWithRole(client, users.UserConfig())
+	require.NoError(t, err)
+	crtbName := namegen.RandStringLower(8)
+
+	inputData := unmarshalReadJson(t, readJson, remoteClusterID, crtbName, newUser.ID, defaultRoleTemplateName)
+	require.NoError(t, err)
+	inputData["serviceAccount"] = "testServiceAccount"
```

Comment:
sorry I don't have much context to CRTB but, why is this supposed to be ignored in the validation/assert step below?